packages/manager/modules/pci/src/components/pci-order-command/pci-order-command.scss

```diff
@@ -0,0 +1,5 @@
+.overloader_height_tail_logs {
+  .tail-logs {
+    height: auto !important;
```

Comment:
Can't we use a bootstrap utility instead ? Or at least find a selector that prevents the use of `!important`

Reply:
Hi @marie-j 
I tried finding a bootstrap utility but could not find one that worked. I defaulted to a selector and removed the !important so it should be better now.